docs/introduction.rst

```diff
@@ -618,7 +618,8 @@ method creates a kind of `defaultdict` like that in the `collections` module.
 
     >>> class DefaultSortedDict(SortedDict):
     ...     def __missing__(self, key):
-    ...         return 0
+    ...         self[key] = 0
+    ...         return self[key]
```

Comment:
```suggestion
    ...         value = 0
    ...         self[key] = value
    ...         return value
```

Reply:
We need to return a reference for the mutable objects, so I wanted to avoid using the name "value" here, but still, it now work the same as the defaultdict, so it should be fine.